framework/BaseYii.php

```diff
@@ -156,6 +156,15 @@ public static function getAlias($alias, $throwException = true)
     }
 
     /**
+     * @param string $alias
+     * @return bool
+     */
+    public static function isAlias($alias)
```

Comment:
what is the use case for this method? There should not be a need to check for this.